### DISCUSSION TITLE
How to set "opted to hire"

### DISCUSSION THREAD

**Discussion by mrSkyGuy** [anmol098/waka-readme-stats#511] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/f68742a9-60a0-4e17-a626-51e8302be030)
How can I set here "Opted to hire"

**Comment by ok-coder1** [comment, marked as answer]
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/about-available-for-hire

Click on your profile picture in the top right corner, then click Settings. Scroll down and find `Jobs profile`. Tick `Available for hire`.
It should look like this
- [X] Available for hire

Then click `Save jobs profile`.
